app/server/models/user.py

```diff
@@ -57,6 +57,9 @@ class User(ManyOrgBase, ModelBase):
     secret = db.Column(db.String())
     _TFA_secret = db.Column(db.String(128))
     TFA_enabled = db.Column(db.Boolean, default=False)
+    _encrypted_pin = db.Column(db.String())
```

Comment:
can we call this `pin_hash` instead? (Because it's hashed rather than encrypted, and it matches the password terminology)